test/e2e_test.go

```diff
@@ -174,7 +185,34 @@ var _ = Describe("e2e", func() {
 				resp, err := channel.CreateChannel(osnURL, block, caCertPool, tlsClientCert)
 				Expect(err).NotTo(HaveOccurred())
 				Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
-
+				if IsBFT == "true" && check {
+					osnURLs := [3]string{"https://localhost:7055", "https://localhost:7057", "https://localhost:7059"}
+					clientCerts := [3]string{
+						"../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer2.example.com/tls/server.crt",
+						"../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer3.example.com/tls/server.crt",
+						"../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer4.example.com/tls/server.crt",
+					}
+					clientKeys := [3]string{
+						"../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer2.example.com/tls/server.key",
+						"../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer3.example.com/tls/server.key",
+						"../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer4.example.com/tls/server.key",
```

Comment:
Since you are providing all 3 values for the slice and not specifying a larger size to be filled with nil values, I don't understand the reasoning behind specifying the size `[3]string` rather than just `[]string`